lib/generators/openstax/rescue_from/install/install_generator.rb

```diff
@@ -1,6 +1,6 @@
 require 'rails/generators'
 
-module OpenStax
+module Openstax
```

Comment:
Can we keep `"OpenStax"`?  Since it is what we use elsewhere.  We probably added an inflection somewhere so rails understands it. e.g. https://github.com/openstax/openstax_api/blob/master/lib/openstax_api.rb#L6